Dockerfile

```diff
@@ -0,0 +1,58 @@
+# The official python:2.7 image no longer receives automatic rebuilds (it's
+# a year old as of October, 2021), so use the latest LTS release of Ubuntu
+# that includes Python 2.7 instead.
+FROM ubuntu:20.04
+
+# Install packages needed to run your application (not build deps).
+RUN set -ex \
```

Comment:
Does one need the `set -ex`? Never seen it in `RUN` statements before

Reply:
I guess it was helpful to me for debugging at some point (and might be in the future), though the `-e` is indeed [redundant](https://stackoverflow.com/a/47143497/166053); I'll take that out.